modules/applications/pages/ui-builder/version-control-with-git/update-local-file-path.adoc

```diff
@@ -0,0 +1,21 @@
+= Update the local file path
+:page-aliases: applications:update-local-file-path.adoc
+:description: For self-hosted instances, Bonita UI Builder clones Git repositories to the local file system associated with the persistent volume inside the Docker container.
+
+{description}
+
+* Difficulty: Beginner
+* Estimated time: 15 minutes
+* Prerequisites: Basic knowledge of Docker
+
+== Instructions
+
+1. Modify the `APPSMITH_GIT_ROOT` environment variable to point to the desired file path.
```

Comment:
How to?

Reply:
I'll add it later today.

Reply:
In fact, this feature is deprecated in Appsmith, so I'll just delete the doc.